src/server.js

```diff
@@ -1,6 +1,9 @@
+import 'source-map-support/register'
```

Comment:
Hmm...is there no easier way to get sourcemaps out of Babel? I thought it was just a flag for the babel command line.